R/utils.R

```diff
@@ -179,3 +179,4 @@ auto_quote <- function(x) {
 
 is_syntactic <- function(x) make.names(x) == x
 has_quotes <- function(x) str_detect(x, "^(`|'|\").*\\1$")
+normalize_quotes <- function(x) str_replace(x, "^(`|'|\")(.*)\\1$", "\\2")
```

Comment:
IMO this is stripping, not normalising. (I'd called it normalising if (e.g.) you were switching `` ` `` and `'` to `"`")